node/pkg/watchers/ibc/watcher.go

```diff
@@ -63,7 +63,13 @@ func setFeatures(f string) {
 
 var (
 	// Chains defines the list of chains to be monitored by IBC. Add new chains here as necessary.
-	Chains = []vaa.ChainID{vaa.ChainIDSei}
+	Chains = []vaa.ChainID{
+		vaa.ChainIDSei,
+		vaa.ChainIDCosmoshub,
```

Comment:
We should change this from Cosmos Hub to Cosmos. Hub is the name of the mainnet and gaia is the name of the testnet. Is it possible to change the sdk at this point still?

Unless we want to add a CosmosGaia as well, but that doesn't fit how most all of the rest of the chains are.

Reply:
@SEJeff not sure your comment is quite correct. It looks to me like both [Mainnet](https://github.com/cosmos/mainnet) and [Testnets](https://github.com/cosmos/testnets) are referred to as Cosmos Hub and [Gaia](https://hub.cosmos.network/main/getting-started/what-is-gaia.html) is the Cosmos SDK application.

Reply:
I stand corrected sir. They've changed this from when I last worked on cosmos. Perfect!